miden-tx/tests/test_miden_faucet_contract.rs

```diff
@@ -254,14 +254,19 @@ fn test_faucet_contract_creation() {
     assert!(faucet_account.is_faucet() == true);
 
     let exp_faucet_account_code_src =
-        include_str!("../../miden-lib/asm/faucets/basic_fungible.masm");
+        include_str!("../../miden-lib/asm/miden/faucets/basic_fungible.masm");
     let exp_faucet_account_code_ast = ModuleAst::parse(exp_faucet_account_code_src).unwrap();
     let mut account_assembler = assembler();
 
     let exp_faucet_account_code =
         AccountCode::new(exp_faucet_account_code_ast.clone(), &mut account_assembler).unwrap();
 
-    assert!(faucet_account.code() == &exp_faucet_account_code);
+    println!("faucet_account.code() = {:?}", faucet_account.code());
+    println!(
+        "exp_faucet_account_code = {:?}",
+        exp_faucet_account_code
+    );
+    //assert!(faucet_account.code() == &exp_faucet_account_code);
```

Comment:
I was incorrect - the test does fail for me to. From what I can tell, the main issue is that when we write the `.masl` library we do not serialize source locations. 

So, the question is: do we want to serialize source locations for `midenlib`?

If yes, then we need to make a small change in `build.rs` file (just pass `true` to `MaslLibrary::read_from_dir()` for the `with_source_locations` parameter).

If no, then we should update the test to remove source locations from the expected account code. This can be done by adding a line to say `exp_faucet_account_code_ast.clear_locations();`.

I'm leaning toward the `yes` option as it may make debugging easier - but maybe this should be something configured via feature flags?

Reply:
Ok that was easy, indeed that works

Reply:
we can do that with feature flags, but why would we not want that as standard? Compiling only happens once, right?